packages/email/email.js

```diff
@@ -43,27 +48,73 @@ var makeTransport = function (mailUrlString) {
     mailUrl.query.pool = 'true';
   }
 
-  var transport = nodemailer.createTransport(
-    urlModule.format(mailUrl));
+  const transport = nodemailer.createTransport(url.format(mailUrl));
+
+  transport._syncSendMail = Meteor.wrapAsync(transport.sendMail, transport);
+  return transport;
+};
+
+// More info: https://nodemailer.com/smtp/well-known/
+const knownHostsTransport = function(settings = undefined, url = undefined) {
+  let service, user, password;
+  if (url && !settings) {
+    let host = url?.split(':')[0];
+    const urlObject = new URL(url);
+    if (host === 'http' || 'https') {
```

Comment:
this is always going to be true, I believe there is a typo here.

Reply:
Not the case if you have url like this: `Outlook365://firstname.lastname:password@hotmail.com`

Reply:
I mean, in Javascript this is always going to be "true":

```js
if (host === 'http' || 'https')
```

as 'https' is always "true".

Reply:
:facepalm: